### PR TITLE
Upgrade terraform-provider-aws to v5.80.0

### LINKED ISSUE #4883: Upgrade terraform-provider-aws to v5.80.0
Release details: https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.80.0

> [!NOTE]
> This issue was created automatically by the upgrade-provider tool and should be automatically closed by a subsequent upgrade pull request.

### PR DESCRIPTION
**Pull request by flostadler**
This PR was generated via `$ upgrade-provider pulumi/pulumi-aws --kind=provider --target-version=5.80.0`.

---

- Upgrading terraform-provider-aws from 5.79.0  to 5.80.0.
	Fixes #4883

### CODE REVIEW

**Comment by flostadler** [comment]
Mhm, this is odd. The new `s3_tables_*` resources get schematized as `s3.Tables*` (e.g. `s3.TablesTable`).
Looking into this

File: patches/0077-Fix-tags_all-Computed-for-CodeConnections-Host-and-C.patch [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+@@ -87,7 +87,7 @@ func (r *hostResource) Schema(ctx context.Context, req resource.SchemaRequest, r
+ 				},
+ 			},
+ 			names.AttrTags:    tftags.TagsAttribute(),
+-			names.AttrTagsAll: tftags.TagsAttributeComputedOnly(),
++			names.AttrTagsAll: tftags.TagsAttribute(),
+ 		},
+ 		Blocks: map[string]schema.Block{
+ 			names.AttrVPCConfiguration: schema.ListNestedBlock{
+diff --git a/tools/tfsdk2fw/go.mod b/tools/tfsdk2fw/go.mod
```
Comment (flostadler) [inline patches/0077-Fix-tags_all-Computed-for-CodeConnections-Host-and-C.patch:33]: Those are unrelated changes... Removing them

**Review by corymhall — APPROVED** [review]
